go-controller/pkg/config/config.go

```diff
@@ -1058,18 +1058,18 @@ func pathExists(path string) bool {
 
 // parseAddress parses an OVN database address, which can be of form
 // "ssl:1.2.3.4:6641,ssl:1.2.3.5:6641" or "ssl://1.2.3.4:6641,ssl://1.2.3.5:6641"
+// or "ssl:[fd01::1]:6641,ssl:[fd01::2]:6641
 // and returns the validated address(es) and the scheme
 func parseAddress(urlString string) (string, OvnDBScheme, error) {
 	var parsedAddress, scheme string
 	var parsedScheme OvnDBScheme
 
 	urlString = strings.Replace(urlString, "//", "", -1)
 	for _, ovnAddress := range strings.Split(urlString, ",") {
-		splits := strings.Split(ovnAddress, ":")
-		if len(splits) != 3 {
+		splits := strings.SplitN(ovnAddress, ":", 2)
```

Comment:
There's actually `net.SplitHostPort`

Reply:
This split is splitting between `protocol` and `host:port`